tests/integrations/raftstore/test_hibernate.rs

```diff
@@ -277,6 +278,9 @@ fn test_split_delay() {
 fn test_inconsistent_configuration() {
     let mut cluster = new_node_cluster(0, 3);
     configure_for_hibernate(&mut cluster);
+    // Sometimes the lease renewing may delay the time of going to hibernate, disable it to
```

Comment:
> lease renewing may delay the time of going to hibernate

Why?

Reply:
Hibernation could be delayed because of pending reads

Reply:
Have you tested the hibernate speed difference between enabling and disabling infinite lease?

Reply:
Don't have an explicit value of hibernate speed, but through metrics of hibernate state, seems there is little difference.

Reply:
But `test_inconsistent_configuration` did become unstable after introducing this PR

Reply:
Seems a bug. Any logs? It passed tens of times locally.

Reply:
It was failed one time at `assert!(!awakened.load(Ordering::SeqCst));` in my environment, I guess that was caused by my changes.